wallet/schemas/de.schildbach.wallet.AppDatabase/19.json

```diff
@@ -1,8 +1,8 @@
 {
   "formatVersion": 1,
   "database": {
-    "version": 18,
-    "identityHash": "5f155ffa8cb6c602b3066210b77c4a55",
+    "version": 19,
+    "identityHash": "b09ba261cdc49ce124f51ac7bf14f299",
```

Comment:
I am not sure why 18 is renamed to 19 instead of having both.

Reply:
Unless 18 is an intermediate state from previous development that is no longer used.  Of course, since we haven't released dashpay in a while it probably doesn't matter anyways.

Reply:
But the migrations only to go 18.

Reply:
Having 19 is a mistake, I'll remove it. Version 19 was only used for development to avoid crashes.
I think we might have duplicated folders for database schemas - will need to figure this out later too.

Reply:
Yes, I also noticed that because schemas were defined in different models?

Reply:
Yes, might be because of my refactoring. AppDatabase is located in a `database` package now.